modules/installation-requirements-user-infra.adoc

```diff
@@ -34,7 +48,10 @@ The smallest {product-title} clusters require the following hosts:
 
 * Three control plane, or master, machines
 
-* At least two compute machines, which are also known as worker machines
+* At least two compute machines, which are also known as worker machines.
+ifdef::bare-metal[]
+Zero compute machines is supported if you are running a three-node cluster. Running one compute machine is not supported.
```

Comment:
Suggest "Running zero compute machines is supported" because otherwise it reads a bit odd (as if it should be "Zero compute machines **are** supported" but that's not really what you mean here).
Also could flip to get the "if" clause up front: "If you are running a three-node cluster, running zero compute machines is supported."